lesson 13/my_car.py

```diff
@@ -0,0 +1,67 @@
+from car import Car
+# Encapsulation ^
```

Comment:
it is not Encapsulation 
Encapsulations it is combining different functions in one general. So user can just call one function instead of 5 for example.